common/src/api/external/mod.rs

```diff
@@ -1275,6 +1275,31 @@ pub enum InstanceAutoRestartPolicy {
     BestEffort,
 }
 
+// AFFINITY GROUPS
+
+#[derive(Clone, Copy, Debug, Deserialize, Serialize, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum AffinityPolicy {
+    Allow,
```

Comment:
I know I'm calling this "Allow" instead of "Alert", but without a fleshed-out alerting system, I wanted to keep this honest.

We can add "Alert" later, when it can actually be implemented.

Reply:
That makes sense to me. Perhaps we should document the behavior here?

Reply:
Done!